packages/react-core/src/components/DualListSelector/DualListSelectorTree.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/DualListSelector/dual-list-selector';
+import { DualListSelectorTreeItem } from './DualListSelectorTreeItem';
+
+export interface DualListSelectorTreeItemData {
+  /** Content rendered inside the dual list selector. */
+  children?: DualListSelectorTreeItemData[];
+  /** Additional classes applied to the dual list selector. */
+  className?: string;
+  /** Flag indicating the option is currently selected. */
+  isSelected?: boolean;
+  /** Flag indicating this option is in the chosen pane. */
+  isChosen?: boolean;
+  /** Flag indicating this option is expanded by default. */
+  defaultExpanded?: boolean;
+  /** Flag indicating this option has a checkbox */
+  hasCheck?: boolean;
+  /** Flag indicating this option has a badge */
+  hasBadge?: boolean;
+  /** Internal callback to pass this ref up to the parent. */
+  sendRef?: (optionRef: React.ReactNode, index: number) => void;
+  /** Internal field used to keep track of the order of filtered options. */
+  filteredIndex?: number;
```

Comment:
I'm removing these props, since the Tree version isn't building a ref array (as of yet). They were holdovers from the initial DualListSelectorItem I worked off of.

Also, only `DualListSelector` is exported so I'm not sure if it matters, since a user can't access the sub-components?

Reply:
@tlabaj Not sure if this applies anymore, but the way to hide props is to start the comment with `@hide` like `/** @hide Internal field consumers need not worry about */`.